### ISSUE TITLE
Rpy2 import error and rpy2 object gets NULL after sometime.

### ISSUE THREAD

**Issue by manibha-jain** [rpy2/rpy2#770]
**Describe the issue or bug**

A clear and concise description of what the issue or bug is.  
-> i am using rpy2 3.4.2  with python 3.6  and R installed with version 4.0.3 and i am calling R script through robjects.r.source function but sometimes this code works and sometimes robjects.r is  NULL. My code is as follows:-

import rpy2.robjects as ro
path2script = os.path.join(settings.BASE_DIR, 'r_scripts', 'script.R')
ro.globalenv['args'] = ro.vectors.StrVector([txt_path])
obj = ro.r.source(path2script)					
returnarray  = np.array(obj, dtype="object")

Please note that this to report demonstrated or suspected issues or bugs with rpy2. Check the project's documentation or Stackoverflow to check whether the answer can be found there.
-> cannot find answer ti my question in listed questions. 
**To Reproduce**

Steps to reproduce the behavior are generally required.
-> Try to run the code again and again so the error generates.
Also import statement produces error following :-
signal.signal(signal.SIGINT, _sigint_handler)

This means a self-contained example (that is an example that contains everything needed to try reproduce the issue).

If this is an issue about installing the software, provide as much information about your system as you think is related: conda used? virtualenv used? environment variables influencing the behavior of R or Python? version for R, Python, and rpy2? Provide also the exact steps followed.

If this is an issue about the rpy2 code, provide all the above, and a minimal code example to trigger the issue. If data is used, that data will be needed to reproduce the issue.

**Expected behavior**

**Error**
```
File "/home/ubuntu/4triangle/frontend/views.py", line 34, in <module>
    import rpy2.robjects as ro
 File "<frozen importlib._bootstrap>", line 971, in _find_and_load
   File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
   File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
   File "/home/ubuntu/4triangle/env/lib/python3.6/site-packages/rpy2/robjects/__init__.py", line 442, in <module>
 r = R()
  File "/home/ubuntu/4triangle/env/lib/python3.6/site-packages/rpy2/robjects/__init__.py", line 402, in __new__
   rinterface.initr_simple()
   File "/home/ubuntu/4triangle/env/lib/python3.6/site-packages/rpy2/rinterface.py", line 859, in initr_simple
    _post_initr_setup()
   File "/home/ubuntu/4triangle/env/lib/python3.6/site-packages/rpy2/rinterface.py", line 940, in _post_initr_setup
   signal.signal(signal.SIGINT, _sigint_handler)
```

**Additional context**
'NULLType' object is not callable -> this error occurs  when ajax is used .

**Comment by krassowski** [comment]
More details in https://stackoverflow.com/questions/65919406/rpy2-in-python-3-6-working-sometimes-and-sometimes-not.

It looks like #769.

**Comment by manibha-jain** [comment]
yeah i saw this link yesterday but it doesn't seem to be solved. What do you think?

> More details in https://stackoverflow.com/questions/65919406/rpy2-in-python-3-6-working-sometimes-and-sometimes-not.
> 
> It looks like #769.

**Comment by lgautier** [comment]
It looks like #769 to me. There is a PR in the works.

**Comment by lgautier** [comment]
PR #784 was merged some time back. Is the issue gone with the latest rpy2 release?

**Comment by manibha-jain** [comment]
It been  a while I have worked on the Rpy2 but what I remember is, this issue isn't solved and my workaround was making a terminal call inside python when the Rpy2 object is null. Which worked for me.
